examples/table/table.html

```diff
@@ -121,6 +121,12 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
           <td><code>div</code></td>
           <td>Refers to the element that serves as the caption for the table.</td>
         </tr>
+        <tr data-test-id="caption-role">
+          <th scope="row"><code>caption</code></th>
+          <td></td>
+          <td><code>div</code></td>
+          <td>The element containing a description of the table contents.</td>
```

Comment:
Suggest wording consistent with other roles

```suggestion
          <td>Identifies the element as a caption for the table.</td>
```